lib/ansible/modules/monitoring/zabbix/zabbix_user.py

```diff
@@ -200,6 +202,7 @@
         default: 'present'
         choices: ['present', 'absent']
         type: str
+        elements: dict
```

Comment:
Type=str and elements=dict ?

Reply:
oh my, thanks for noticing. What a horrible typo